script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -533,7 +534,7 @@ The wind blows fiercely, and with a piercing howl envelops the city in absolute
 [sha:3dad0829f90c4d33437fed7e650d50f20c35a3a8]{
 -- Page 552, Offset 10085.
 -- そして、
-And then.
+And then...
```

Comment:
I think general style we've been using is that commas in cases like this are replaced by three em dashes, since that's more of a truncation (event happening suddenly in next line[s]) and an ellipsis is more of a trailing off